### PR TITLE
Use transport API without Vercel rewrite

### PR DESCRIPTION
**Pull request by sashachabin**
I checked, the production build works.

And the requests are sent (if the server has not crashed, as it is now) :)

- [x] Removed Vercel `/api` rewrite
- [x] Removed `http-proxy-middleware` for local dev (now we use `next dev`)
- [x] Added `Access-Control-Allow-Origin` header
     - prod: `transport.ekaterinburg.city`
     - dev: `localhost`
- [x] Added env variable `NODE_ENV=production` for API server

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import express from 'express';
-
 import { router } from './router/index';
+import { addCorsHeaders } from './utils/add-cors-headers';
 
 const app = express();
 const port = process.env.APP_PORT || 3080;
 
+
+app.use(addCorsHeaders);
```
Comment (hiba9201) [inline server/index.ts:9]: why not cors middleware for express? https://www.npmjs.com/package/cors

**Review by hiba9201 — APPROVED** [review]

File: server/index.ts (hunk 1 above)
Comment (sashachabin) [reply server/index.ts:9]: I was scared of this 👽👽👽

image

Comment (sashachabin) [reply server/index.ts:9]: https://www.npmjs.com/package/cors

Comment (sashachabin) [reply server/index.ts:9]: And old dependencies:
https://github.com/expressjs/cors/blob/master/package.json#L16

image

**Pull request merged**